cowprotocol/cms-legacy
# Dockerise CMS

Adds dockerisation for the CMS.

CMS is a Strapi instance created out of https://docs.strapi.io/dev-docs/intro.

# How to Test?
* Copy .env.example to .env
* Add values required & save
* Run `docker compose up`
* You should be able to get the application up and running at `localhost:1337`.
